lib/cli.ex

```diff
@@ -19,11 +23,14 @@ defmodule Onigumo.CLI do
     components = Enum.join(Map.keys(@components), ", ")
 
     IO.puts("""
-    Usage: onigumo [COMPONENT]
+    Usage: onigumo [OPTION]... [COMPONENT]
 
     Simple program that retrieves HTTP web content as structured data.
 
     COMPONENT\tOnigumo component to run, available: #{components}
+
+    OPTIONS:
+    -C, --working-dir=DIR\tChange to <DIR> before processing remaining files
```

Comment:
The short version does not use `=`, but the long one may. It may be confusing to use it in the usage example, because it makes the impression that `-C=<dir>` is a valid usage. However, wget does it exactly like this and apparently it’s not a problem. 😃

cURL, on the other hand, does not use `=`, but a space. It also uses `<lowercase>` notation for the variables and does not mention the variable in the description. Like this:

```suggestion
    -C, --working-dir <dir>\tChange working directory before processing remaining files
```

For completeness, _tar_ has it like this:

```
-C <dir>  Change to <dir> before processing remaining files
```

I don’t have any strong preference. Just mentioning.

Reply:
I prefer the way how the cURL do it, because my experience that typing with **space** is more common in reality then **=**.